Source/EventHorizon/Consumer/EventProcessorPolicy.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Dolittle.Logging;
+using Dolittle.Resilience;
+using Dolittle.Runtime.Events.Store;
+using Polly;
+
+namespace Dolittle.Runtime.EventHorizon.Consumer
+{
+    /// <summary>
+    /// Defines the policy for processing an event from an event horizon.
+    /// </summary>
+    public class EventProcessorPolicy : IDefineAsyncPolicyForType
+    {
+        readonly ILogger<EventProcessor> _logger;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="EventProcessorPolicy"/> class.
+        /// </summary>
+        /// <param name="logger">The <see cref="ILogger"/> to use for logging.</param>
+        public EventProcessorPolicy(ILogger<EventProcessor> logger)
+        {
+            _logger = logger;
+        }
+
+        /// <inheritdoc/>
+        public Type Type => typeof(EventProcessor);
+
+        /// <inheritdoc/>
+        public Polly.IAsyncPolicy Define()
+            => Polly.Policy
+                .Handle<EventStoreUnavailable>(
```

Comment:
Don't we want to catch all kinds of exceptions here?
If this policy returns without retrying, what happens to the event horizon subscription?

Reply:
Makes sense - as long as the subscription is restarted elsewhere (as it is) then we don't need to retry other exceptions here.

The last one was probably a glitch in my brain, I was thinking that exceptions that weren't handled here would dissapear. But I assume then that anything other than `EventStoreUnavailable` will bubble out from the policy, and then be handled elsewhere.